en/rocky/8/admin_guide/01-presentation.md

```diff
@@ -0,0 +1,299 @@
+
+Linux equips a little more than **3% of personal computers** but more than **82% of smartphones**. **Android** being an operating system whose kernel is a Linux.
+<!-- TODO: review those stats -->
+
+Linux equips 100% of the 500 supercomputers since 2018. A supercomputer is a computer designed to achieve the highest possible performance with the techniques known at the time of its design, especially with regard to computing speed.
+
+### Architectural design
+
+* The **kernel** is the first software component.
+ * It is the heart of the UNIX system.
+ * It manages the hardware resources of the system.
+ * The other software components must go through it to access the hardware.
+* The **shell** is a utility that interprets user commands and ensures their execution.
+ * Main shells: Bourne shell, C shell, Korn shell and Bourne-Again shell (bash).
+* Applications are user programs such as :
+ * Internet browser ;
+ * the word processor ;
+ * ...
+
+#### Multitask
+
+Linux belongs to the family of time-sharing operating systems. It shares process time between several programs, switching from one to another in a transparent way for the user. This implies:
+
+* simultaneous execution of several programs;
+* distribution of CPU time by the scheduler;
+* reduction of problems due to a failed application;
+* reduced performance when there are too many programs running.
+
+#### Multi-users
+
+The purpose of Multics was to allow several users to work from several terminals (screen and keyboard) on a single computer (very expensive at the time). Linux, which is inspired by this operating system, has kept this ability to work with several users simultaneously and independently, each one having its own user account, memory space and access rights to files and software.
+
+#### Multiprocesseur
+
+Linux is able to work with multi-processor computers or with multi-core processors.
+
+#### Multiprocessor
+
+Linux is written in a high-level language that can be adapted to different types of platforms during compilation. It therefore runs on :
+
+* home computers (PC or laptop);
+* servers (data, applications,...);
+* portable computers (smartphones or tablets)
+* embedded systems (car computer);
+* active network elements (routers, switches)
+* household appliances (TVs, refrigerators,...).
+
+#### Open
+
+Linux is based on recognized standards [posix](http://fr.wikipedia.org/wiki/POSIX), TCP/IP, NFS, Samba ... allowing to share data and services with other application systems.
+
+### The UNIX phylosophy
+
+* Everything is a file.
+* Portability.
+* Do only one thing and do it well.
+* KISS: Keep It Simple and Stupid.
+* " UNIX is basically a simple operating system, but you have to be a genius to understand the simplicity." (__Dennis Ritchie__)
+* "Unix is user-friendly. It just isn't promiscuous about which users it's friendly with." (__Steven King__)
+
```

Comment:
Love these quotes! Made me laugh anyway! :-)

Reply:
Me too. You are a genius you know ^^